spec/rubocop/cop/style/send_spec.rb

```diff
@@ -0,0 +1,43 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::Send do
+  subject(:cop) { described_class.new }
+
+  context 'with send' do
+    after(:each) { expect(cop.offenses.size).to eq(1) }
+
+    it 'registers an offense for a single expression' do
```

Comment:
What's this obsession with the `single expression`? :-) How about "a send invocation" or something like this?